clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderProducerCallback.java

```diff
@@ -124,6 +126,15 @@ public void onCompletion(PubSubProduceResult produceResult, Exception e) {
                 ingestionTask.versionNumber,
                 LatencyUtils.getLatencyInMS(produceTimeNs),
                 currentTimeForMetricsMs);
+        if (ingestionTask.isHybridMode() && sourceConsumerRecord.getTopicPartition().getPubSubTopic().isRealTime()
```

Comment:
Shall we avoid reporting for system stores too, just like the above metric?

Reply:
The system store check on line 122 applies to this metric as well.